lnbits/core/views/admin_api.py

```diff
@@ -27,6 +31,7 @@
 from ..crud import delete_admin_settings, get_admin_settings, update_admin_settings
 
 admin_router = APIRouter(tags=["Admin UI"], prefix="/admin")
+file_upload = File(...)
```

Comment:
why is this defined outside the method params?

Reply:
ruff